src/js/oTable.js

```diff
@@ -19,16 +19,22 @@ function OTable(rootEl) {
 	if (this.rootEl !== undefined) {
 		this.listeners = [];
 		this.isResponsive = false;
-		this.rootEl.setAttribute('data-o-table--js', '');
+		this.rootEl.setAttribute('data-o-table-js', '');
 
-		this.tableHeaders = Array.from(this.rootEl.querySelectorAll('thead th'));
 		const tableRows = Array.from(this.rootEl.getElementsByTagName('tr'));
 
-		this.tableHeaders.forEach((th, columnIndex) => {
-			const listener = this._sortByColumn(columnIndex);
-			this.listeners.push(listener);
-			th.addEventListener('click', listener);
-		});
+		// Get tables headers from the last header row (not headers of headers).
+		this.tableHeaders = Array.from(this.rootEl.querySelectorAll('thead tr:last-of-type th'));
+		const firstDataRow = this.rootEl.querySelectorAll('tbody tr:first-of-type td');
```

Comment:
May need to break this querySelectorAll into multiple queries -- https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll#User_notes